src/rules/at-rule-blockless-newline-after/__tests__/index.js

```diff
@@ -0,0 +1,42 @@
+import {
+  ruleTester,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule("always", tr => {
+  tr.ok("@import 'x.css';\na {}")
+  tr.ok("@charset 'UTF-8';\na {}")
+  tr.ok("@namespace url(XML-namespace-URL);\na {}")
+  tr.ok("@import 'x.css'); /* comment */\n")
+  tr.ok("@import 'x.css');/* comment */\n")
+  tr.ok("@import 'x.css');   /* comment */\n")
+  tr.ok("@import 'x.css');\t/* comment */\n")
+  tr.ok("@import 'x.css'); \t/* comment */\n")
+  tr.ok("@import 'x.css';\r\n", "windows")
+  tr.ok("@import 'x.css'; /* comment */\r\n", "windows")
+  tr.ok("@import 'x.css';", "single blockless rule")
```

Comment:
None of the above have a rule or at-rule after the initial at-rule. Could we add that?